index.js

```diff
@@ -10,10 +14,18 @@ module.exports = (api, options) => {
         '--headless': 'Run in headless mode',
         '--mode': 'Loads a corresponding .env.[mode] file',
         '--gever': 'Use a real GEVER backend',
-        '--spec': 'Specify which test file(s) to run instead of running all tests'
+        '--spec': 'Specify which test file(s) to run instead of running all tests',
+        '--runserver': `Also start the backend (with Django's runserver) (default: ${defaults["runserver"]})`
       },
     },
-    async (args) => { await start(api, Object.assign(options, args)); }
+    async (args) => {
+      for (const key in defaults) {
```

Comment:
You could use `Object.assign(args, defaults)` as long as the object is not nested.

Reply:
👍 - updated.

I copied this from https://github.com/vuejs/vue-cli/blob/master/packages/%40vue/cli-service/lib/commands/build/index.js#L44-L48 ...